PatientRegistrationApp/src/components/ScreeningQuestions.vue

```diff
@@ -0,0 +1,320 @@
+              <v-radio-group v-model="screeningQ6" row required>
+                <v-radio label="Yes" value="Yes"></v-radio>
+                <v-radio label="No" value="No"></v-radio>
+              </v-radio-group>
+            </v-row>
+          </v-col>
+        </v-row>
+        <v-row no-gutters>
+          <v-divider></v-divider>
+        </v-row>
+        <v-row no-gutters>
+          <v-col cols="12" sm="8" md="9" lg="10">
+            <v-row no-gutters>
+              <v-radio-group>
+                <template v-slot:label>
+                  <div><span class="red--text"><strong>* </strong></span>7. Are you currently under isolation or quarantine due to COVID-19?</div>
+                </template>
+              </v-radio-group>
+            </v-row>
+          </v-col>
+          <v-col cols="12" sm="4" md="3" lg="2">
+            <v-row no-gutters align="center" justify="end">
+              <v-radio-group v-model="screeningQ7" row required>
+                <v-radio label="Yes" value="Yes"></v-radio>
+                <v-radio label="No" value="No"></v-radio>
+              </v-radio-group>
+            </v-row>
+          </v-col>
+        </v-row>
+        <v-row no-gutters>
+          <v-divider></v-divider>
+        </v-row>
+        <v-row no-gutters>
+          <v-col cols="12" sm="8" md="9" lg="10">
+            <v-row no-gutters>
+              <v-radio-group>
+                <template v-slot:label>
+                  <div><span class="red--text"><strong>* </strong></span>8. Have you received passive antibody therapy or convalescent plasma as treatment for COVID-19 in the past 90 days?</div>
+                </template>
+              </v-radio-group>
+            </v-row>
+          </v-col>
+          <v-col cols="12" sm="4" md="3" lg="2">
+            <v-row no-gutters align="center" justify="end">
+              <v-radio-group v-model="screeningQ8" row required>
+                <v-radio label="Yes" value="Yes"></v-radio>
+                <v-radio label="No" value="No"></v-radio>
+              </v-radio-group>
+            </v-row>
+          </v-col>
+        </v-row>
+        <v-row no-gutters>
+          <v-divider></v-divider>
+        </v-row>
+      </v-col>
+    </v-row>
+	</v-form>
+  </v-container>
+</template>
+
+<script>
+import EventBus from '../eventBus'
+
+export default {
+  name: "ScreeningQuestions",
+  data() {
+    return {
+		screeningQ1: '',
+		screeningQ2: '',
+		screeningQ2b: '',
+		screeningQ3a: '',
+		screeningQ3b: '',
+		screeningQ3c: '',
+		screeningQ4: '',
+		screeningQ5: '',
+		screeningQ6: '',
+		screeningQ7: '',
+		screeningQ8: '',
+		patientReceivedFirstDose: false,
+    };
+  },
+  methods: {
+	FirstDoseResponse() {
+		this.patientReceivedFirstDose = (this.screeningQ2 == 'Yes')
+		if(!this.patientReceivedFirstDose) {
+			this.screeningQ2b = ''
+		}
+	},
+    sendScreeningResponsesToFollowUpPage() {
+      const screeningResponsesPayload = {
+        screeningQ1: this.screeningQ1,
```

Comment:
The naming here just makes me feel weird. Can we not capture all of this in an array or something? Referring to them as screeningQ## is super vague. What happens if we change the order of the questions? The data fields might not match things down the line?

Reply:
I think we will need to come back to this once we learn how screening questions might vary by county.  I'm assuming we will need to make these customer-configurable, so that's why I went with the non-specific screeningQ# naming rather than a more tailored name to match what the question is asking.  Once we know more about what other counties will want to ask in terms of screening questions, I'm expecting that we would refine this implementation more.  For example, there will probably be some of the questions that become more "fixed" (like asking about the manufacturer of dose 1), but then there will probably be some questions that vary slightly by wording (as it seems each county might have different lawyers review the wording).

I do not disagree with your comment - but I do think we need to work through this with at least 1 more county/city to figure out the most appropriate implementation approach.